src/scripts.js

```diff
@@ -52,22 +54,37 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
+function createDropdownTags() {
```

Comment:
Nice job building out this function with iterator methods. Good use of correct syntax.